CITATION

```diff
@@ -0,0 +1,15 @@
+Morris, B.D. and E.P. White. 2013. The EcoData Retriever: improving access to
+existing ecological data. PLOS ONE 8:e65848.
+http://doi.org/doi:10.1371/journal.pone.0065848
+
+@article{morris2013ecodata,
+  title={The EcoData Retriever: Improving Access to Existing Ecological Data},
+  author={Morris, Benjamin D and White, Ethan P},
+  journal={PLOS One},
+  volume={8},
+  number={6},
+  pages={e65848},
+  year={2013},
+  publisher={Public Library of Science}
+  doi={10.1371/journal.pone.0065848}
+}
```

Comment:
Let's get rid of the CITATION file for the moment. We can address this later once the project is more fully formed.